geomstats/_backend/pytorch/logm.py

```diff
@@ -0,0 +1,36 @@
+"""
+Pytorch Differentiable Matrix Logarithm
+Implementation based on:
+https://github.com/pytorch/pytorch/issues/9983#issuecomment-891777620
+"""
+import torch
+from ..numpy import linalg as nplinalg
+
+
+def _adjoint(A, E, f):
```

Comment:
Thank you for this! Let's just make this code PEP8 compliant: https://realpython.com/python-pep8/

i.e.:
- to uppercase letters for variables of function parameters: uppercase are reserved from class names: A -> mat_a, E -> mat_e
- avoid single-letter variables (it's just a nightmare to do automatic find and replace)